app/grants/views.py

```diff
@@ -1585,12 +1585,18 @@ def grant_new(request):
                 'id': g_category.pk,
                 'name': g_category.category
             })
-        grant_types.append({
+
+        grant_type_temp = {
             'id': g_type.pk,
             'name': g_type.name,
             'label': g_type.label,
             'categories': grant_categories
-        })
+        }
+        if g_type.logo:
+            grant_type_temp['image_url'] = request.build_absolute_uri(g_type.logo.url)
+
+        grant_types.append(grant_type_temp
```

Comment:
This seems broken

Reply:
@octavioamu did pitch the idea of sending all the banners to the frontend and having the frontend decide which type to use! 

Reason being: if i the matic logo is present and I switch from to filecoin -> shouldn't the image change as well ?

Reply:
Yes, if the logo for filecoin is available, it would change. Otherwise, it uses the default icon.